go-apps/meep-app-enablement/server/service-mgmt/service-mgmt.go

```diff
@@ -0,0 +1,654 @@
+	isLocal           string
+	scopeOfLocality   string
+}
+
+/*func notImplemented(w http.ResponseWriter, r *http.Request) {
+        w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+        w.WriteHeader(http.StatusNotImplemented)
+}
+*/
+func Init() (err error) {
+	// Retrieve Sandbox name from environment variable
+	sandboxNameEnv := strings.TrimSpace(os.Getenv("MEEP_SANDBOX_NAME"))
+	if sandboxNameEnv != "" {
+		sandboxName = sandboxNameEnv
+	}
+	if sandboxName == "" {
+		err = errors.New("MEEP_SANDBOX_NAME env variable not set")
+		log.Error(err.Error())
+		return err
+	}
+	log.Info("MEEP_SANDBOX_NAME: ", sandboxName)
+
+	// hostUrl is the url of the node serving the resourceURL
+	// Retrieve public url address where service is reachable, if not present, use Host URL environment variable
+	hostUrl, err = url.Parse(strings.TrimSpace(os.Getenv("MEEP_PUBLIC_URL")))
+	if err != nil || hostUrl == nil || hostUrl.String() == "" {
+		hostUrl, err = url.Parse(strings.TrimSpace(os.Getenv("MEEP_HOST_URL")))
+		if err != nil {
+			hostUrl = new(url.URL)
+		}
+	}
+	log.Info("resource URL: ", hostUrl)
+
+	selfNameEnv := strings.TrimSpace(os.Getenv("MEEP_SELF_NAME"))
+	if selfNameEnv != "" {
+		selfName = selfNameEnv
+	}
+	//TODO
+	/*	if selfName == "" {
+			err = errors.New("MEEP_SELF_NAME env variable not set")
+			log.Error(err.Error())
+			return err
+		}
+	*/
+	//HARDCODE SOMETHING
+	selfName = "mep1"
+	log.Info("MEEP_SELF_NAME: ", selfName)
+
+	// Set base path
+	basePath = "/" + sandboxName + msmgmtBasePath
+	// Get base store key
+	baseKey = dkm.GetKeyRoot(sandboxName) + msmgmtKey
+	appEnablementBaseKey = dkm.GetKeyRoot(sandboxName) + selfName + ":" + appEnablementKey
+
+	// Connect to Redis DB
+	rc, err = redis.NewConnector(redisAddr, MSMGMT_DB)
+	if err != nil {
+		log.Error("Failed connection to Redis DB. Error: ", err)
+		return err
+	}
+
+	_ = rc.DBFlush(baseKey)
+	_ = rc.DBFlush(appEnablementBaseKey)
+
+	log.Info("Connected to Redis DB")
+
+	reInit()
+
+	expiryTicker = time.NewTicker(time.Second)
+	go func() {
+		for range expiryTicker.C {
+			//checkForExpiredSubscriptions()
+		}
+	}()
+	/*
+	   // Initialize SBI
+	   sbiCfg := sbi.SbiCfg{
+	           SandboxName:    sandboxName,
+	           RedisAddr:      redisAddr,
+	           InfluxAddr:     influxAddr,
+	           StaInfoCb:      updateStaInfo,
+	           ApInfoCb:       updateApInfo,
+	           ScenarioNameCb: updateStoreName,
+	           CleanUpCb:      cleanUp,
+	   }
+
+	   err = sbi.Init(sbiCfg)
+	   if err != nil {
+	           log.Error("Failed initialize SBI. Error: ", err)
+	           return err
+	   }
+	   log.Info("SBI Initialized")
+	*/
+	return nil
+}
+
+// reInit - finds the value already in the DB to repopulate local stored info
+func reInit() {
+	//next available subsId will be overrriden if subscriptions already existed
+	nextSubscriptionIdAvailable = 1
+	nextServiceRegistrationIdAvailable = 1
+}
+
+// Run - Start WAIS
```

Comment:
Fix comments & remove unused comments (unless necessary for future implementation)

Reply:
fixed